server/datastore/mysql/aggregated_stats.go

```diff
@@ -166,3 +168,19 @@ func walkIdsInTable(
 	}
 	return nil
 }
+
+// GetAggregatedStats retrieves aggregated stats for the given query
+func (ds *Datastore) GetAggregatedStats(ctx context.Context, aggregate fleet.AggregatedStatsType, id uint) (fleet.AggregatedStats, error) {
```

Comment:
if this is only used in tests, I think best to move it to our `test` package, ie somewhere in `server/test/`

Reply:
moved to `testing_utils.go`